src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -50,7 +50,7 @@ def schedule_auto_resolution():
                    soft_time_limit=60)
 def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000,
                                 **kwargs):
-    project = Project.objects.get_from_cache(id=project_id)
```

Comment:
should probably bind tenants in tasks in the same vein as zeus does

e.g. it grabs the first object (unrestricted), and then scopes a tenant based on that objects access pattern